### PR TITLE
added client

### PR DESCRIPTION
**Pull request by vsaveliev**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,77 @@
+all: push
+
+BUILDTAGS=
+
+# Use the 0.0.0 tag for testing, it shouldn't clobber any release builds
+APP?=github-integration
+CHARTS?=mycharts
```
Comment (takama) [inline Makefile]: CHARTS?=charts

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+USERSPACE?=k8s-community
+RELEASE?=0.1.0
```
Comment (takama) [inline Makefile]: RELEASE?=0.0.0
it should be the same as release tag, no tags - no releases

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+PROJECT?=github.com/${USERSPACE}/${APP}
+HELM_REPO?=https://${USERSPACE}.github.io/${CHARTS}
```
Comment (takama) [inline Makefile]: HELM_REPO?=https://services.k8s.community/${CHARTS}

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+GOOS?=linux
+REGISTRY?=registry.k8s.community
+MYAPP_SERVICE_PORT?=8080
```
Comment (takama) [inline Makefile]: There should be your ENV(s) not myapp

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+container: build
+	docker build --pull -t $(PREFIX):$(RELEASE) .
+
+push: container
+	docker push $(PREFIX):$(RELEASE)
+
+run: container
+	docker run --name ${CONTAINER_NAME} -p ${MYAPP_SERVICE_PORT}:${MYAPP_SERVICE_PORT} \
+		-e "MYAPP_SERVICE_PORT=${MYAPP_SERVICE_PORT}" \
```
Comment (takama) [inline Makefile]: There should be your ENV(s)

**Review by takama — CHANGES_REQUESTED** [review]
Please change Makefile in connection with your service specific ENVs

File: Dockerfile [hunk 6]
```diff
@@ -0,0 +1,18 @@
+FROM scratch
+
+ENV GITHUBINT_SERVICE_PORT 8080
+ENV GITHUBINT_BRANCH "workshop/"
+ENV GITHUBINT_TOKEN "Webhook secret is in integration settings on Github"
+ENV GITHUBINT_PRIV_KEY "Private key is in integration settings on Github"
+ENV GITHUBINT_INTEGRATION_ID "Integration ID is in it's settings on Github"
+
+ENV CICD_SERVICE_HOST "localhost"
```
Comment (rumyantseva) [inline Dockerfile]: Давайте лучше `0.0.0.0`, а не `localhost`?

File: Makefile [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+container: build
+	docker build --pull -t $(PREFIX):$(RELEASE) .
+
+push: container
+	docker push $(PREFIX):$(RELEASE)
+
+run: container
+	docker run --name ${CONTAINER_NAME} -p ${GITHUBINT_SERVICE_PORT}:${GITHUBINT_SERVICE_PORT} \
+		-e "MYAPP_SERVICE_PORT=${GITHUBINT_SERVICE_PORT}" \
```
Comment (rumyantseva) [inline Makefile]: Не `MYAPP_SEVICE_PORT`, а `GITHUBINT_SERVICE_PORT `

**Pull request merged**